src/css-grids/style.css

```diff
@@ -0,0 +1,107 @@
+* {
+  box-sizing: border-box;
+  background-color: rgb(242,179,180);
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+h1 {
+  color: white;
+  background-color: black;
+  font-size: 260px;
+  margin-top: 20px;
+  grid-row: 3/5;
+  grid-column: 9/11;
+  z-index: 5;
+}
+
+p {
+  background-color: white;
+  padding: 20px 40px 0px 40px;
+}
+
+h2 {
+  background-color: white;
+  color: rgb(211, 44, 86);
+  text-align: right;
+  padding-right: 40px;
+  padding-bottom: 20px;
+  font-size: 22px;
+}
+
+.gridcontainer { 
+  height: 100%;
+  display: grid;
+  grid-template-columns: 5% repeat(5, 1fr) 5% 2% 11% repeat(2, 13%) 5%;
+  grid-template-rows: 5% 20% 1fr 1fr 30% 5%;
+}
+
+.box {
+  height: 100%;
+  width: 100%;
+}
+
+.red-box { 
+  background: url(headphones.jpg);
```

Comment:
use `background-image: url('./headphones.jpg');`

the file needs to be in quotes.